### PR TITLE
planning using the default plugins directory

### PR DESCRIPTION
**Pull request by eoliveros**
- [x] remove in config the non-default plugins directory
- [x] create the default plugins directory if not existed
- [x] copy the plugins from /opt/lightningd_plugins to the default plugins directory

### CODE REVIEW

File: lightningd/entrypoint-shim.sh [hunk 1]
```diff
@@ -8,5 +8,12 @@ if [ -n "$LIGHTNINGD_REPLICA" ]; then
 fi
 
 jinja2 /config.j2 > "${LIGHTNINGD_DATA}/config"
+if [ -d ${LIGHTNINGD_DATA}/plugins ]
```
Comment (djpnewton) [inline lightningd/entrypoint-shim.sh:11]: probably simpler just to remove this bit and modify the dockerfile from:

`COPY ./rebalance.py /opt/lightningd_plugins/`
to
`COPY ./rebalance.py $LIGHTNINGD_DATA/plugins`

**Review by djpnewton — CHANGES_REQUESTED** [review]

File: lightningd/entrypoint-shim.sh (hunk 1 above)
Comment (eoliveros) [reply lightningd/entrypoint-shim.sh:11]: im sure i've tried this awhile back. but didn't work. its like the config situation where its either empty, not being created or its being forgotten. either way the file being COPY over doesnt exists when the container starts.

Comment (eoliveros) [reply lightningd/entrypoint-shim.sh:11]: yep just confirmed it... its looking like adding a file using COPY from dockerfile doesn't copy the file to the ${LIGHTNINGD_DATA}/plugins directory. even when the plugins directory exists(which doesn't appear to be the case initially).

Comment (djpnewton) [reply lightningd/entrypoint-shim.sh]: ok I figured out why we cant modify $LIGHTNINGD_DATA in the dockerfile and documented it

**Pull request merged**